style.css

```diff
@@ -23,13 +23,13 @@ body {
 
 .headline {
   display: flex;
-  background-image: url("./illustration.png");
+  background-image: url("./img/illustration.png");
```

Comment:
- [ ] [OPTIONAL] - This also had the same issue of the wrong file path naming. Below I will include for you the correct file path

```
  background-image: url("./img/Illustration.png");
```